x-pack/snapshot-tool/qa/build.gradle

```diff
@@ -0,0 +1 @@
+group = "${group}.plugins.snapshot-tool.qa"
```

Comment:
I don't really think this is necessary. We've never going to publish these subprojects. We can ditch this entire file. Gradle projects do not require a build file.

Reply:
With this file removed I get the following error:
```
Could not determine the dependencies of task ':x-pack:snapshot-tool:check'.
> Task with path 'qa:s3:check' not found in project ':x-pack:snapshot-tool'
```